docs/building_chains.rst

```diff
@@ -42,3 +42,41 @@ Then to initialize, you can start it up with an in-memory database:
 
   # initialize a fresh chain
   chain = chain_class.from_genesis_header(chaindb, MAINNET_GENESIS_HEADER)
+
+
+Using the LightChain object
+---------------------------
+
+The :class:`~evm.p2p.lightchain.LightChain` is like a Chain but it will also
+connect to remote peers and fetch new :class:`~evm.rlp.headers.BlockHeader` s
+as they are announced on the network. As such, it must first be configured
+with a vm_configuration, but it also requires a network_id and privkey:
+
+::
+
+  from evm.chains.mainnet import MAINNET_VM_CONFIGURATION, MAINNET_NETWORK_ID
+  from evm.p2p import ecies
+  from evm.p2p.lightchain import LightChain
+
+  DemoLightChain = LightChain.configure(
+      name='Demo LightChain',
+      privkey=ecies.generate_privkey(),
```

Comment:
doesn't need to be done here, but I think it's ok for us to start storing some data under `~/.py-evm/` which would include generating this value automatically for each chain if it's not already present.  This is how other clients do it yes?

Reply:
Created #198